terraform/shared/shared.tf.json.template.py

```diff
@@ -40,24 +40,28 @@ def conformance_pack(name: str) -> str:
 
 
 cis_alarms = [
```

Comment:
```suggestion
trail_alarms = [
```

as part of the commit that fixes the URLs.